pkg/grub/rootfs.cfg

```diff
@@ -338,12 +338,17 @@ function set_arm64_baremetal {
       set_global dom0_console "console=tty0 earlycon=tty0"
       set_global dom0_platform_tweaks "fbcon=map:0 eve_nuke_disks=mmcblk0 eve_install_disk=nvme0n1 video=efifb:off"
    fi
-   # Jetson Orin Nano Developer Kit
+   # Jetson Orin Nano Developer Kit (Jetpack 5.x)
    if [ "$smb_product" = "NVIDIA Orin Nano Developer Kit" ]; then
       set_to_existing_file devicetree /boot/dtb/nvidia/tegra234-p3767-0003-p3768-0000-a0.dtb
-      set_global dom0_console "console=ttyAMA0,115200 console=tty0 console=ttyTCU0,115200 earlycon=ttyTCU0,115200"
+      set_global dom0_console "console=ttyAMA0,115200 console=tty1 console=ttyTCU0,115200 earlycon=ttyTCU0,115200"
```

Comment:
> This commit adds the corresponding entry on GRUB config

and explicitly switches the console option for the jp5 variant to `tty1`. I kind of remember why it's necessary, but I'm a little confused that we still use tty0 and tty1 in different variants...

Reply:
it's necessary because of the TUI monitor... actually we need to update all references for `tty0` for any other platform, but it's outside the scope of this PR... I can open a new one to take care of other platforms....